crates/extrinsics/Cargo.toml

```diff
@@ -32,13 +32,13 @@ serde_json = "1.0.111"
 url = { version = "2.5.0", features = ["serde"] }
 rust_decimal = "1.33"
 tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
-sp-core = "22.0.0"
-sp-runtime = "25.0.0"
-sp-weights = "21.0.0"
-pallet-contracts-primitives = "25.0.0"
+sp-core = "28.0.0"
+sp-runtime = "31.0.0"
+sp-weights = "27.0.0"
+pallet-contracts-uapi = "5.0.0"
```

Comment:
Do we need these dependency? I am not sure if we are currently using any types that require it